pkg/userdata/bottlerocket/node_userdata.go

```diff
@@ -106,11 +118,33 @@ func generateBottlerocketNodeUserData(bootstrapContainerUserData []byte, users [
 	}
 	b64AdminContainerUserData := base64.StdEncoding.EncodeToString(adminContainerUserData)
 
+	hostContainers := []etcdbootstrapv1.BottlerocketHostContainer{
+		{
+			Name:         "admin",
+			Superpowered: true,
+			Image:        config.BottlerocketConfig.AdminImage,
```

Comment:
Oh I see, it is being decoupled here. Which is why we need AdminImage at a higher level separately now?

Reply:
yup